src/main/scala/gitbucket/core/model/activity/comment/CommentActivityInfo.scala

```diff
@@ -0,0 +1,95 @@
+package gitbucket.core.model.activity.comment
+
+import java.util.UUID
+
+import gitbucket.core.model.Activity
+import gitbucket.core.model.Profile.currentDate
+import gitbucket.core.model.activity.BaseActivityInfo
+
+sealed trait CommentActivityInfo extends BaseActivityInfo {
+  def userName: String
+  def repositoryName: String
+  def activityUserName: String
+  def comment: String
+  def maybeIssueId: Option[Int]
+  def maybeCommitId: Option[String]
```

Comment:
Why do we need to have these `maybeXxxx` methods? It looks no one refer to them.

Reply:
See https://github.com/gitbucket/gitbucket/pull/2526#discussion_r478525708